src/FFMpeg/Filters/Video/ExtractMultipleFramesFilter.php

```diff
@@ -50,6 +50,9 @@ public function __construct($frameRate = self::FRAMERATE_EVERY_SEC, $destination
         $this->destinationFolder = $destinationFolder;
     }
 
+	/**
+	 * @param string $frameFileType
+	 */
     public function setFrameFileType($frameFileType) {
     	$this->frameFileType = $frameFileType;
```

Comment:
okay, please check whether `$frameFileType` is one of the extensions defined in a class property with an array of `['jpg', 'jpeg', 'png']`. (`in_array` will help you.)

Reply:
Added together with a test